packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -492,9 +492,32 @@ export class ChannelCollection
 				continue;
 			}
 
-			// If a non-local operation then go and create the object, otherwise mark it as officially attached.
+			// Check for collision with local (not yet live / known to other clients) DataStore
+			// This is not a DataCorruption case if we crash the container before it attaches (it's a DataProcessingError instead)
+			//
+			// POSSIBLE CAUSES:
+			// - Something with ID creation, e.g. a bug in shortID logic, or somehow an alias is used that matches a generated ID
```

Comment:
`an alias is used that matches a generated ID` - I don't think that's possible; at least it won't hit this code but hit the alias message process code.